build/bin/.functions.sh

```diff
@@ -50,7 +50,40 @@ function install_yq() {
   python -m pip install yq || exit 1
 }
 
-
+# Install buildx
+# -----------------------------------------------------------------------------
+# Useful links:
+# - https://docs.docker.com/engine/reference/commandline/buildx_build/#output
+# - https://docs.docker.com/build/building/multi-platform/
+# - https://medium.com/@artur.klauser/building-multi-architecture-docker-images-with-buildx-27d80f7e2408
+# - https://stackoverflow.com/questions/65365797/docker-buildx-exec-user-process-caused-exec-format-error
+function install_buildx() {
+ # docker buildx use mybuilder &> /dev/null
+  echo "install_buildx"
+ # if [ "$?" != "0" ]; then
+     echo "inside0 install_buildx"
+    mkdir -vp ~/.docker/cli-plugins/
+    #https://download.docker.com/linux/rhel/8/x86_64/stable/Packages/docker-buildx-plugin-0.16.1-1.el8.x86_64.rpm
+    curl --silent -L "https://github.com/docker/buildx/releases/download/v0.11.2/buildx-v0.11.2.linux-amd64" > ~/.docker/cli-plugins/docker-buildx
+    chmod a+x ~/.docker/cli-plugins/docker-buildx
+
+    echo "inside 1 install_buildx"
+    sudo apt-get update
+    echo "inside 1-1 install_buildx"
+    sudo apt-get install -y qemu-user-static
+      echo "inside2 install_buildx"
+    qemu-aarch64-static --version
+echo "inside3 install_buildx"
+    sudo apt-get install -y binfmt-support
+    update-binfmts --version
+
+    docker buildx create --name mybuilder
+    docker buildx use mybuilder
+#  fi
```

Comment:
Indenting looks wrong here and we have a commented out if block, that should be deleted if not used